app/models/manageiq/providers/datawarehouse_manager.rb

```diff
@@ -0,0 +1,13 @@
+module ManageIQ::Providers
+  class DatawarehouseManager < BaseManager
+    class << model_name
+      def route_key
+        "ems_datawarehouse"
```

Comment:
@martinpovolny @himdel It seems to only works this way, I'm not what are the implications...